demos/continuous_batching/README.md

```diff
@@ -195,22 +230,53 @@ for chunk in stream:
 
 Output:
 ```
-This is a test.
+It looks like you're testing me!
+```
+
+A similar code can be applied for the completion endpoint:
+```bash
+pip3 install openai
+```
+```python
+from openai import OpenAI
+
+client = OpenAI(
+  base_url="http://localhost:8000/v3",
+  api_key="unused"
+)
+
+stream = client.completions.create(
+    model="meta-llama/Meta-Llama-3-8B-Instruct",
+    prompt="<|begin_of_text|><|start_header_id|>user<|end_header_id|>\n\nSay this is a test.<|eot_id|><|start_header_id|>assistant<|end_header_id|>",
+    stream=True,
+)
+for chunk in stream:
+    if chunk.choices[0].text is not None:
+        print(chunk.choices[0].text, end="")
+```
+
+Output:
+```
+It looks like you're testing me!
```

Comment:
Is it worth putting a raw text here? It doesn't show streaming.

Reply:
Documentation to be addressed in next PRs after RC